tests/providers/cncf/kubernetes/log_handlers/test_log_handlers.py

```diff
@@ -0,0 +1,169 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+from __future__ import annotations
+
+import logging
+import logging.config
+import re
+from importlib import reload
+from unittest import mock
+from unittest.mock import patch
+
+import pytest
+from kubernetes.client import models as k8s
+
+from airflow.config_templates.airflow_local_settings import DEFAULT_LOGGING_CONFIG
+from airflow.executors import executor_loader
+from airflow.models.dag import DAG
+from airflow.models.dagrun import DagRun
+from airflow.models.taskinstance import TaskInstance
+from airflow.operators.python import PythonOperator
+from airflow.utils.log.file_task_handler import (
+    FileTaskHandler,
+)
+from airflow.utils.log.logging_mixin import set_context
+from airflow.utils.session import create_session
+from airflow.utils.state import State, TaskInstanceState
+from airflow.utils.timezone import datetime
+from airflow.utils.types import DagRunType
+from tests.test_utils.compat import AIRFLOW_V_3_0_PLUS
+from tests.test_utils.config import conf_vars
+
+if AIRFLOW_V_3_0_PLUS:
+    from airflow.utils.types import DagRunTriggeredByType
+
+pytestmark = [pytest.mark.db_test, pytest.mark.skip_if_database_isolation_mode]
+
+DEFAULT_DATE = datetime(2016, 1, 1)
+TASK_LOGGER = "airflow.task"
+FILE_TASK_HANDLER = "task"
+
+
+class TestFileTaskLogHandler:
+    def clean_up(self):
+        with create_session() as session:
+            session.query(DagRun).delete()
+            session.query(TaskInstance).delete()
+
+    def setup_method(self):
+        logging.config.dictConfig(DEFAULT_LOGGING_CONFIG)
```

Comment:
Somehow I fear - but am not sure - that the chang eof log config influences the TestDbApiHook tests which now fail in missing logs :-(